.github/workflows/zeebe-commitlint.yml

```diff
@@ -1,5 +1,5 @@
+name: Zeebe Commitlint
```

Comment:
💭 I think this is not really Zeebe related. This is more a project specific which would apply to all right?

Reply:
Probably true yea, this is more in the scope of "(human) processes and workflows in the future monorepo" (like issue management) :+1:

I will remove it.